easybuild/easyconfigs/g/GPAW/GPAW-0.11.0-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '0.11.0'
```

Comment:
this version is quite old, any reason for using this rather than the 1.2.0 available at https://pypi.python.org/pypi/gpaw?